docs/index.html

```diff
@@ -56,7 +56,8 @@ <h2>Get started now</h2>
       <div class="steps">
         <div class="step step-1">
           <!--label>Step 1</label-->
-          <a class="console-btn" href="https://console.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/cloud-ops-sandbox.git&cloudshell_git_branch=v0.2.5&cloudshell_working_dir=terraform&shellonly=true&cloudshell_image=gcr.io/stackdriver-sandbox-230822/cloudshell-image:v0.2.5" target="_blank" onclick="metrics.recordCustomTime('stackdriver-demo-open-in-cloud-shell')">Open in Google Cloud Shell</a>
+          <a class="console-btn" href="https://console.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/cloud-ops-sandbox.git&cloudshell_git_branch=v0.2.5&cloudshell_working_dir=terraform&shellonly=true&cloudshell_image=gcr.io/stackdriver-sandbox-230822/cloudshell-image:v0.2.5"
+             target="_blank" onclick="if (confirm('Stackdriver Sandbox will create a new GCP project and will take ~15-20 minutes to complete. Are you sure you want to continue?')) { metrics.recordCustomTime('stackdriver-demo-open-in-cloud-shell');  return true; } else { return false; } ">Open in Google Cloud Shell</a>
```

Comment:
Cloud Ops Sandbox.
Also remove _metrics.Recordtime_ statement

Reply:
It's a little strange since the website says Stackdriver Sandbox everywhere else for now. I was thinking we should stick with Stackdriver in the popup to be consistent and change everything at once later? But I can make that change now too

Are we removing the metrics from this page then?

Reply:
At this point the only thing that says "Stackdriver" should be the address (stackdriver-sandbox.dev). Everything else should be renamed. We'll get a new dns record later too, so the last mention will go away

Reply:
Should I make the other changes to the website as part of this PR?

Reply:
would be great if you can